src/master-playlist-controller.js

```diff
@@ -1350,6 +1346,9 @@ export class MasterPlaylistController extends videojs.EventTarget {
     if (usingAudioLoader && unsupportedAudio && this.media().attributes.AUDIO) {
       const audioGroup = this.media().attributes.AUDIO;
 
+      this.mediaTypes_.AUDIO.activePlaylistLoader.pause();
```

Comment:
This is important because if we wait for the masterPlaylistLoader to `mediachange` and then call `abort` then we will already have appended unsupported codecs.